// Copyright 2016 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package cc.proto;

// Control messages sent to the main side of the compositor(server) from the
// impl side of the compositor(client).
message CompositorMessageToMain {
  // TODO(khushalsagar): Add messages sent to the main compositor from the impl
  // side.
}
